Tests/Unit/Collector/FormatterTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Http\HttplugBundle\Tests\Unit\Collector;
+
+use Http\Client\Exception\HttpException;
+use Http\Client\Exception\TransferException;
+use Http\HttplugBundle\Collector\Formatter as CollectorFormatter;
+use Http\Message\Formatter;
```

Comment:
could we alias this to MessageFormatter and not alias the formatter we are testing, for better readability?